stubs/pywin32/isapi/__init__.pyi

```diff
@@ -1,10 +1,10 @@
 from _typeshed import Incomplete
 
 class ISAPIError(Exception):
-    errno: Incomplete
-    strerror: Incomplete
+    errno: int
+    strerror: str | None
     funcname: Incomplete
```

Comment:
It's made to look like a `win32api.error`. From tests we can see other errors' `funcname` should be `str | None`.

```suggestion
    funcname: str | None
```